setup.py

```diff
@@ -192,7 +208,7 @@ def check_linker_need_libatomic():
 if EXTRA_ENV_LINK_ARGS is None:
     EXTRA_ENV_LINK_ARGS = ''
     if "linux" in sys.platform or "darwin" in sys.platform:
-        EXTRA_ENV_LINK_ARGS += ' -lpthread'
+        EXTRA_ENV_LINK_ARGS += ' -ldl -lpthread'
```

Comment:
Will `-ldl` impact our compatibility across platforms? We can check it by running the Fusion job: https://fusion.corp.google.com/projectanalysis/summary/KOKORO/prod%3Agrpc%2Fcore%2Fexperimental%2Fgrpc_build_artifacts_multiplatform.

Reply:
I do not think it will. Note that it is only added on Linux and Darwin.

BTW, how can I run the Fusion job?

Reply:
There is a TRIGGER BUILD button.

Reply:
The fusion test passed: https://g3c.corp.google.com/results/invocations/b6251e7b-13c1-49a7-806a-ba4279a1c375/targets/grpc%2Fcore%2Fexperimental%2Fgrpc_build_artifacts_multiplatform/log